### ISSUE TITLE
Fly binary downloads stop working after a while

### ISSUE THREAD

**Issue by itsdalmo** [concourse/concourse#2626]
## Bug Report

Downloading `fly` binaries from the Concourse web interface stops working when `concourse web` has been running for an undetermined period of time.

## Steps to Reproduce

Launch the binary version of `concourse web` and leave it running for an extended period of time; eventually the `fly` downloads will stop working. I can't say exactly how long it has to be left running before it stops working, unfortunately.

## Expected Results

A `fly` binary should be downloaded.

## Actual Results

When attempting to download a binary, Chrome fails:
```
This site can’t be reached
The web page at https://concourse.example.com/api/v1/cli?arch=amd64&platform=darwin might be temporarily down or it may have moved permanently to a new web address.
ERR_INVALID_RESPONSE
```

Whereas Safari actually downloads a file `fly.txt` with the following content:
```
404 page not found
```

## Additional Context

Restarting `concourse web` resolves the issue.

## Version Info

* Concourse version: `v3.14.1`
* Deployment type (BOSH/Docker/binary): binary
* Infrastructure/IaaS: AWS EC2 (VM)
* Browser (if applicable): Chrome/Safari
* Did this used to work?: Not as far as I can recall.

**Comment by pms1969** [comment]
I've noticed this too.  Just assumed it was unique to us, and had something to do with our setup.  A restart usually fixes it, and a fly sync still works (iirc).

**Comment by itsdalmo** [comment]
Closing this as it seems to be fixed after upgrading to `5.1.0`, I assume because the fly assets are part of the distribution now.
